packages/components/table/src/table.vue

```diff
@@ -66,6 +66,7 @@
           :wrap-style="scrollbarStyle"
           :always="scrollbarAlwaysOn"
           :tabindex="scrollbarTabindex"
+          style="text-align: left"
```

Comment:
It is better to write this static style directly in the style file.
https://github.com/element-plus/element-plus/blob/dev/packages/theme-chalk/src/table.scss